docs/how_to/point_manipulator.md

```diff
@@ -8,28 +8,29 @@ Precisely positioning a point in 3D is a useful way to annotate specific positio
 
 ## Instructions
 
-1. Open the [points_manipulator.py](https://napari-threedee.github.io/generated/gallery/plugin/points_manipulator_plugin/) example.
+1. To run this example, first download the script from the [Examples Gallery page](https://napari-threedee.github.io/generated/gallery/plugin/points_manipulator_plugin/); the link is at the bottom. Ensure the location is somewhere memorable. Or, if you've cloned this repository, the script can be found at `napari-threedee/docs/examples/plugin/points_manipulator_plugin.py`. 
 
+2. Ensure you have the appropriate virtual environment active, change directories to the script location, and then you can execute the script using:
+	```bash
+	python points_manipulator_plugin.py
+	``` 
 
-    ![example opened](https://user-images.githubusercontent.com/1120672/207381513-8fd3f7e9-157f-4ef9-99c1-551bd329842b.png)
-   The example opens in 3D rendering mode. Click and drag to rotate the view.
-   
-2. Open the point manipulator plugin from the Plugins menu: Plugins -> napari-threedee -> point manipulator. Click the "activate" to start the plugin.
+3. The viewer will open in 3D rendering mode with a Points layer and 3 points and the widget already visible. The widget can be otherwise accessed from the Plugins menu: **Plugins -> napari-threedee -> point manipulator**. Click the "activate" to start the plugin.
 
 	![plugin opened](https://user-images.githubusercontent.com/1120672/207382282-dad2bd6f-68cf-47d6-89ed-be326d320f93.png)
 
-4. Set the Points later to "Selection" mode by clicking the "Selection" button in the layer controls
+4. Set the Points later to "Selection" mode by clicking the "Selection" button in the layer controls.
 
 	![point selection mode](https://user-images.githubusercontent.com/1120672/207382556-9cd2111a-1a01-4102-9de8-4bb87ddad3c3.png)
 
-5. Select a point by clicking on it. The point manipulator will appear on the selected point.
+5. Select a point by clicking on it. The point manipulator will appear on the selected point. Note that depending on your versions of `napari-threedee` and `napari` you may see subtle visual differences between your viewer and the screenshots/videos here.
```

Comment:
Is this sentence really necessary? Can we remove it?
*"Note that depending on your versions of `napari-threedee` and `napari` you may see subtle visual differences between your viewer and the screenshots/videos here."*

Reply:
I think we should either keep it or make a note somewhere else in the docs. We did update the appearance of the manipulators since these screenshots were created. I think the usage doesn't really change, so we don't have to completely update the screenshots.

My vote for now is to keep it here.

Reply:
There are differences in the napari UI too, different/extra tool icons.
My idea was to avoid re-do'ing all the visuals -- the videos in particular. 
I just don't have the time for that.